tests/unit/sagemaker/jumpstart/constants.py

```diff
@@ -26,13 +26,123 @@
         },
         "hosting_artifact_key": "pytorch-infer/infer-pytorch-eqa-bert-base-cased.tar.gz",
         "hosting_script_key": "source-directory-tarballs/pytorch/inference/eqa/v1.0.0/sourcedir.tar.gz",
+        "inference_vulnerable": False,
+        "inference_dependencies": [
+            "transformers==3.5.1",
+            "dataclasses==0.8",
+            "filelock==3.0.12",
+            "packaging==20.8",
+            "pyparsing==2.4.7",
+            "regex==2020.11.13",
+            "sacremoses==0.0.43",
+            "sentencepiece==0.1.91",
+            "tokenizers==0.9.3",
+        ],
+        "inference_vulnerabilities": [],
+        "training_vulnerable": False,
+        "training_dependencies": [
+            "transformers==3.5.1",
+            "dataclasses==0.8",
+            "filelock==3.0.12",
+            "packaging==20.8",
+            "pyparsing==2.4.7",
+            "regex==2020.11.13",
+            "sacremoses==0.0.43",
+            "sentencepiece==0.1.91",
+            "tokenizers==0.9.3",
+        ],
+        "training_vulnerabilities": [],
+        "deprecated": False,
+        "hyperparameters": [
+            {
+                "name": "epochs",
+                "type": "int",
+                "default": 3,
+                "min": 1,
+                "max": 1000,
+                "scope": "algorithm",
+            },
+            {
+                "name": "adam-learning-rate",
+                "type": "float",
+                "default": 2e-05,
+                "min": 1e-08,
+                "max": 1,
+                "scope": "algorithm",
+            },
+            {
+                "name": "batch-size",
+                "type": "int",
+                "default": 4,
+                "min": 1,
+                "max": 1024,
+                "scope": "algorithm",
+            },
+            {
+                "name": "sagemaker_submit_directory",
+                "type": "text",
+                "default": "/opt/ml/input/data/code/sourcedir.tar.gz",
+                "scope": "container",
+            },
+            {
+                "name": "sagemaker_program",
+                "type": "text",
+                "default": "transfer_learning.py",
+                "scope": "container",
+            },
+            {
+                "name": "sagemaker_container_log_level",
+                "type": "text",
+                "default": "20",
+                "scope": "container",
+            },
+        ],
         "training_script_key": "source-directory-tarballs/pytorch/transfer_learning/eqa/v1.0.0/sourcedir.tar.gz",
         "training_ecr_specs": {
             "framework_version": "1.5.0",
             "framework": "pytorch",
             "py_version": "py3",
         },
         "training_artifact_key": "pytorch-training/train-pytorch-eqa-bert-base-cased.tar.gz",
+        "inference_environment_variables": [
```

Comment:
I see same value for `inference_environment_variables` used multiple times in this file, can we create a variable and help remove redundancy.

Reply:
Sorry, I'd prefer to explicitly list the specs here, taken straight from metadata. This will mean that many of the fields may look repeated or redundant.